### PR TITLE
[camera] Apply new external texture APIs

### PR DESCRIPTION
**Pull request by bbrto21**
Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

File: packages/camera/tizen/src/camera_device.cc [hunk 1]
```diff
@@ -326,7 +324,40 @@ CameraDevice::CameraDevice(flutter::PluginRegistrar *registrar,
   SetResolutionPreset(resolution_preset_);
 
   // Init channels
-  texture_id_ = FlutterRegisterExternalTexture(texture_registrar_);
+  // texture_id_ = FlutterRegisterExternalTexture(texture_registrar_);
```
Comment (bwikbs) [inline packages/camera/tizen/src/camera_device.cc:327]: :scissors:!

File: packages/camera/tizen/src/camera_device.cc [hunk 2]
```diff
@@ -852,6 +882,27 @@ void CameraDevice::UpdateStates() {
   GetRecorderState(recorder_state_);
 }
 
+bool CameraDevice::RegistAvailableMediaPacket(media_packet_h packet) {
+  std::lock_guard<std::mutex> lock(mutex_);
+  if (packet_) {
```
Comment (bwikbs) [inline packages/camera/tizen/src/camera_device.cc:887]: It's not much of a difference, but how about replacing it with a new packet?

File: packages/camera/tizen/src/camera_device.h [hunk 3]
```diff
@@ -323,9 +321,16 @@ class CameraDevice {
   bool UnsetRecorderRecordingLimitReachedCb();
   void UpdateStates();
 
+  bool RegistAvailableMediaPacket(media_packet_h packet);
+  media_packet_h UnregistAvailableMediaPacket();
```
Comment (swift-kim) [inline packages/camera/tizen/src/camera_device.h:325]: Regist → Register?

File: packages/camera/tizen/src/camera_device.cc (hunk 1 above)
Comment (bbrto21) [reply packages/camera/tizen/src/camera_device.cc]: Thanks!

File: packages/camera/tizen/src/camera_device.cc (hunk 2 above)
Comment (bbrto21) [reply packages/camera/tizen/src/camera_device.cc]: please review again, I updated this PR

File: packages/camera/tizen/src/camera_device.h (hunk 3 above)
Comment (bbrto21) [reply packages/camera/tizen/src/camera_device.h]: it is no longer a member, please review again

**Review by swift-kim — APPROVED** [review]

**Pull request merged**